packages/dds/tree/src/core/tree/anchorSet.ts

```diff
@@ -564,21 +573,50 @@ export class AnchorSet implements ISubscribable<AnchorSetRootEvents> {
 			}
 		};
 
+		/**
```

Comment:
Comments on locals should just be `//` comments, since TS-Doc don't look inside functions.